deploy/ingress/nginx-ingress-values.yaml

```diff
@@ -2,6 +2,10 @@
 ## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
 ##
 controller:
+  admissionWebhooks:
+    enabled: false
+    patch:
+     enabled: false
```

Comment:
Would be great to understand what this options do. Update of the ref about would help it I guess. Please fix the reference with the link to up-to-date documentation you used for this change

Reply:
@ytsarev this is something new chart adds, to prevent new features piled up, I just disable them to keep things as they were. In a nutshell, this prevents validation webhook being installed: https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/templates/admission-webhooks/validating-webhook.yaml so far it supports networking.k8s.io only

Reply:
> so far it supports networking.k8s.io only

@k0da, does it mean we should merge it after #417 ?

Reply:
I won't pull it into this PR. We can enable it later. So IMO this is not a blocker